src/middleware.ts

```diff
@@ -1,13 +1,15 @@
 import createIntlMiddleware from 'next-intl/middleware';
 
+export const locales = ['en', 'nl'];
```

Comment:
This makes sense. Thinking out load, though: there could be a mismatch between the locales defined here and the locales defined in the locale files (in `messages`): we have two places with locales. Is there a way to have just one place, to be absolutely sure about the source of truth? For example: can the middleware (or next-intl) detect the locale files?

Reply:
Thanks for looking into this. Could you add a comment to the file, to make this clear (as a note to self, and for future developers)?

Reply:
Ok, I have added a comment.

Reply:
Excellent! 👍🏻